src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DoNotDirectlyAwaitATask.Fixer.cs

```diff
@@ -71,8 +78,26 @@ private class MyCodeAction : DocumentChangeAction
                 base(title, createChangedDocument, equivalenceKey)
             {
             }
+        }
 
-            public override string EquivalenceKey => Title;
+        private sealed class CustomFixAllProvider : DocumentBasedFixAllProvider
```

Comment:
Is this change related to this PR (support 'await using')?

Reply:
Yes, the test was not able to produce correct results for the Fix All cases without this change.